presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/serde/GeometrySerializationType.java

```diff
@@ -68,4 +68,28 @@ public static GeometrySerializationType getForCode(int code)
                 throw new IllegalArgumentException("Invalid type code: " + code);
         }
     }
+
+    public String toString()
```

Comment:
Why do we need this? Why the standard toString from the enum is not good enough? If this is something that we use for actual serialization / deserialization, should we have a separate method and not reuse the `toString` interface?

Reply:
This is used for a user-facing error message -- it's a bit nicer and conforms to the previous error message.  However, I have no strong feelings and I'm happy for users to read `MULTI_LINE_STRING`.

Reply:
I've removed it: it's probably better to change the test error message than add a new method that must be maintained.